kakarot_scripts/deployment/main.py

```diff
@@ -92,6 +93,14 @@ async def main():
         f"💰  EVM deployer balance:\n    L2: {l2_balance} ETH\n    L1: {l1_balance} ETH"
     )
 
+    # check precompiles received funds
+    for i in range(1, 10):
```

Comment:
I would not loop to 10 but make sure _all_ precompiles even non eth ones are warmed

can check my pending PR on warmup to see how the python vars have changed

Reply:
Done